src/test/java/org/apache/aurora/scheduler/TaskStatusHandlerImplTest.java

```diff
@@ -233,7 +234,7 @@ public void testSuppressUnregisteredExecutorMessage() throws Exception {
         TASK_ID_A,
         Optional.absent(),
         KILLED,
-        Optional.absent()))
+        Optional.of("Service not Healthy")))
```

Comment:
This means that executor_unregistered implies health checks failing in this example?

Reply:
I thought about only setting that if the task has a healthcheck configured..